sdk/lib/rtl/slist.c

```diff
@@ -132,10 +132,10 @@ RtlInterlockedPushListSList(
             NewSListHead.Header16.Init = 1;
 
             /* Atomically exchange the SlistHead with the new one */
-            exchanged = _InterlockedCompareExchange128((PULONG64)SListHead,
+            exchanged = _InterlockedCompareExchange128((PLONG64)SListHead,
```

Comment:
https://learn.microsoft.com/en-us/cpp/intrinsics/interlockedcompareexchange128?view=msvc-170

Signed version looks reasonable.

Reply:
All interlocked functions for numerical values, act on pointer to signed values, anyway.